addons/io.md

```diff
@@ -7,7 +7,13 @@ title: System Integrations
 
 # System Integrations
 
-openHAB supports services that enable integration with various technologies that don't fall into other add-on categories.
+openHAB supports services that enable integration with various technologies
+that don't fall into other add-on categories.
+
+Most of the below mentioned systems are integrated via a *Misc* add-on
+installed through Paper UI.
+Detailed instructions and possible requirements can be found
+in the corresponding documentation pages.
```

Comment:
Please consider, "Detailed instructions and configuration options may be found in the corresponding add-on documentation pages."

Is that right - the "possible requirements" you are talking about are the options for how to set up the service - right?

Reply:
>Is that right - the "possible requirements" you are talking about are the options for how to set up the service - right?

Yeah like you need to use the cloud connector for setting up the alexa integration.
That is a "requirement" for me.
And its called requirement in the corresponding docs.

See:
https://docs.openhab.org/addons/ios/alexa-skill/readme.html#requirements

Reply:
Yes - okay with “requirements “.

Reply:
i would remove the possible after reading through that sentence again.
If there are any requirements one will notice them, sicn we are pointing to the corresponding documentaion with that sentence anyway.

Reply:
Ah and one thing:

I would not use "add-on configuration pages".
Alexa for example is not an add-on itself.

Reply:
👍